src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -71,6 +71,18 @@
             <argument type="service" id="templating.locator" />
         </service>
 
+        <service id="twig.extension.profiler" class="Symfony\Bridge\Twig\Extension\ProfilerExtension" public="false">
+            <argument type="service" id="twig.profile" />
+            <argument type="service" id="debug.stopwatch" on-invalid="ignore" />
```

Comment:
``on-invalid="ignore"`` is invalid because the argument is mandatory in the class